src/dotnet/APIView/APIViewWeb/Models/PackageGroupMdel.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System.Collections.Generic;

```

Comment:
Looks like there is a typo in this file name. Should it be `PackageGroupModel.cs`? Should we fix it?